R/plot_compare_stage.R

```diff
@@ -1,10 +1,9 @@
-#'@title Plot simulated and observed stage data
+#'Plot simulated and observed stage data
```

Comment:
Is this suggested syntax, or is it just a fancy way to do the documentation? As in, is there advantage to dropping the @title tag?